bitsail-connectors/connector-hbase/src/main/java/com/bytedance/bitsail/connector/hbase/HBaseHelper.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2022-2023 Bytedance Ltd. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.hbase;
+
+import com.bytedance.bitsail.common.util.Preconditions;
+import com.bytedance.bitsail.connector.hbase.auth.KerberosAuthenticator;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.client.BufferedMutator;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.ConnectionFactory;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.security.PrivilegedAction;
+import java.util.Map;
+import java.util.Objects;
+
+public class HBaseHelper implements AutoCloseable {
+  private static final Logger LOG = LoggerFactory.getLogger(HBaseHelper.class);
+
+  /**
+   * Get connection with hbase.
+   * @param hbaseConfigMap HBase configuration.
+   */
+  public static Connection getHbaseConnection(Map<String, Object> hbaseConfigMap) {
```

Comment:
Can we use normal functions instead of static functions?

Reply:
Sure. Changed it to non static functions.